libs/design/src/core/sticky/daffStickyTracker-test.html

```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
```

Comment:
I don't see the `daffStickyTracker` directive being used in this demo. Are you just duplicating the directive code in the script?

If so, the demo needs to be moved to `design-land` so you can demo the `DaffStickyTrackerDirective` in a custom component.